src/api/app/views/webui2/webui/package/_files_view.html.haml

```diff
@@ -27,8 +27,10 @@
                 = link_to(file_url(project.name, package.name, file[:name], file[:srcmd5]), title: 'Download file') do
                   %i.fas.fa-arrow-circle-down.text-secondary
               - if removable_file?(file_name: file[:name], package: package) && User.current.can_modify?(package)
-                = link_to({ action: :remove_file, project: project.to_param, package: package.to_param, filename: file[:name] },
-                  data: { confirm: "Really remove file '#{file[:name]}'?" }, method: :post, title: 'Remove file') do
+                = render(partial: 'delete_file_dialog',
+                         locals: { project: project.to_param, package: package.to_param, filename: file[:name], file_id: index })
+                = link_to('#', data: { toggle: 'modal', target: "#delete-file-modal-#{index}" },
+                          title: 'Remove file') do
```

Comment:
Again, consistency: `Delete file`

Reply:
Again, is consistent with the text we had before

Reply:
I point this out in the multiple views when it happens just to be sure we don't forget it :slightly_smiling_face: